libcaf_core/caf/catch_all.hpp

```diff
@@ -16,18 +16,17 @@ template <class F>
 struct catch_all {
   using fun_type = std::function<skippable_result(message&)>;
 
-  static_assert(std::is_convertible_v<F, fun_type>,
-                "catch-all handler must have signature "
-                "skippable_result (message&)");
+  static_assert(
+    std::is_convertible_v<F, fun_type>,
+    "catch-all handler must have signature skippable_result (message&)");
```

Comment:
Why did you change the formatting?

Reply:
Must've been a mistake and clang-tidy didn't return it the way it was before.